chia/cmds/data.py

```diff
@@ -263,10 +272,20 @@ def get_root_history(
     run(get_root_history_cmd(rpc_port=data_rpc_port, store_id=id))
 
 
-@data_cmd.command("add_missing_files", short_help="")
-@click.option("-i", "--ids", help="", type=str, required=False)
-@click.option("-o/-n", "--override/--no-override")
-@click.option("-f", "--foldername", type=str, required=False)
+@data_cmd.command("add_missing_files", short_help="Manually reconstruct server files from the data layer database")
+@click.option(
+    "-i",
+    "--ids",
+    help="List of stores to reconstruct. If not specified, all stores will be reconstructed",
+    type=str,
+    required=False,
+)
+@click.option(
+    "-o/-n", "--override/--no-override", help="Specify if already existing files need to be overridden by this command"
```

Comment:
'replace' might be a generally better word to consider here in the future?

```suggestion
    "-o/-n", "--override/--no-override", help="Specify if already existing files need to be overwritten by this command"
```